src/reducers/settings.js

```diff
@@ -85,6 +86,7 @@ const INITIAL_STATE: SettingsState = {
   readOnlyModeEnabled: !Config.DISABLE_READ_ONLY,
   experimentalUSBEnabled: false,
   countervalueFirst: false,
+  developerMode: !!__DEV__,
```

Comment:
mmh I don't think this is what we want. the developerModeEnabled is actually mapped to `getEnv("DEV_MODE")`, which is **not** the same as __DEV__. it's basically a user's settings that enable or not the testnet apps and currencies.

Reply:
(it's ok to call getEnv("DEV_MODE") at runtime from the selector because it's going to be set when you enter the screen)

Reply:
@gre This is what we do on desktop though... https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/reducers/settings.js#L88

Reply:
I think that `developerMode` is a dead code unfortunately! seems like it was supposed to get dropped as we move it all to the DEV_MODE env. it's used in bunch of place in live-common and is the way we filter apps and currencies.

Reply:
the question is do we restore it back correctly, i think we don't. i wonder if we want to really use that in the "export" if it's not really working. We might not want to export the experimental settings. 

wdyt @aramab  ?

Reply:
Export is definitely using that, I followed the selector there, so if there's changes to make there I'll port them over to desktop too :)

Reply:
I would export it IMO